test/omit-spec.js

```diff
@@ -11,4 +11,10 @@ describe('u.omit', () => {
   it('freezes the result', () => {
     expect(Object.isFrozen(u.omit('a', {}))).to.be.true
   })
+
+  it("doesn't change the obj if nothing is omitted", () => {
+    const orig = {a: 1}
```

Comment:
Style is still wrong on these, please ensure it passes prettier/lint.

Reply:
Weird, `$ npx prettier --write test/omit-spec.js ` is happy with that block. Are you using a different invocation?

Reply:
I just pushed an update to the prettier config. It was inconsistent with what the code was. `yarn format` should format now. I also reformatted everything in the repo accordingly (specifically types, please take a look at them and see if you have any concerns).

Thanks!

Reply:
Ugh, dtslint wants semicolons. I'll just exclude the types.

Reply:
But if you can rebase this on master and update formatting w/ `yarn format` that would be appreciated. Sorry for all the hoops on this, it was wrong in the repo before ☹️

Reply:
Done!